src/docs/product/alerts/index.mdx

```diff
@@ -44,6 +44,10 @@ You can find a full list of available metric alerts in [Metric Alerts](/product/
 
 When you create a new project in [sentry.io](https://sentry.io), you can select a default issue alert. However, you can also [create your own alerts](/product/alerts/create-alerts/) to suit your team’s needs, using these [best practices](/product/alerts/best-practices/) as a guide.
 
+## Muting Alerts
+
+Issue alerts can be muted on the alert rule details page by clicking the "Mute" button. This will mute the alert from notifying you until you click "Unmute". If you want to mute the alert from firing entirely, select "Mute for everyone" from the dropdown.
```

Comment:
@ceorourke Might want to mention the required permission for `mute for everyone`

Reply:
Hmm looks like it was changed to anyone w/ `alerts:write` which is .... [members and up](https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1995)? Is that right? That doesn't seem good.

Reply:
we changed it to alerts:write since that's the permission you need to create/delete alerts. if they can delete an alert, it seems like muting is a lesser version of that (especially since it tracks who muted in the banner so people can see)

Reply:
@ceorourke So it depends on the org. Some orgs let members create alerts, some orgs don't. It's an organization setting. Richard has the right idea: if you have the permission to delete an alert you should definitely have permission to mute.

Reply:
😬 okay I'm on board w/ matching that, separate conversation about whether that permission is a good idea lol. I wouldn't want some random person on another team to be able to delete an alert I depend on. Regardless, do we need to mention permissions when there is no restriction?

Reply:
Oh I just saw this banner: https://docs.sentry.io/product/alerts/create-alerts/ I can copy that

Reply:
@ceorourke yes exactly!

Reply:
The wording seems wrong on that banner though, "The minimum role required to create alerts is member. " when actually it's only member if you toggle on that option ... shouldn't it be admin?

Reply:
@ceorourke the default now is to allow members to create alerts so it's only if they explicitly turn it off (or they are a legacy org). I'll leave the language up to you.